internal/testcafe/config.go

```diff
@@ -0,0 +1,111 @@
+package testcafe
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/internal/config"
+	"gopkg.in/yaml.v2"
+)
+
+// Project represents the testcafe project configuration.
+type Project struct {
+	config.TypeDef `yaml:",inline"`
+	ShowConsoleLog bool
+	Sauce          config.SauceConfig `yaml:"sauce,omitempty" json:"sauce"`
+	Suites         []Suite            `yaml:"suites,omitempty" json:"suites"`
+	BeforeExec     []string           `yaml:"beforeExec,omitempty" json:"beforeExec"`
+	Docker         config.Docker      `yaml:"docker,omitempty" json:"docker"`
+	Testcafe       Testcafe           `yaml:"testcafe,omitempty" json:"testcafe"`
+	Npm            config.Npm         `yaml:"npm,omitempty" json:"npm"`
+}
+
+// Suite represents the testcafe test suite configuration.
+type Suite struct {
+	Name               string            `yaml:"name,omitempty" json:"name"`
+	BrowserName        string            `yaml:"browserName,omitempty" json:"browserName"`
+	BrowserVersion     string            `yaml:"browserVersion,omitempty" json:"browserVersion"`
+	Src                []string          `yaml:"src,omitempty" json:"src"`
+	Screenshots        Screenshots       `yaml:"screenshots,omitempty" json:"screenshots"`
+	PlatformName       string            `yaml:"platformName,omitempty" json:"platformName"`
+	ScreenResolution   string            `yaml:"screenResolution,omitempty" json:"screenResolution"`
+	Env                map[string]string `yaml:"env,omitempty" json:"env"`
+	TsConfigPath       string            `yaml:"tsConfigPath,omitempty" json:"tsConfigPath"`
+	ClientScripts      []string          `yaml:"clientScripts,omitempty" json:"clientScripts"`
+	SkipJsErrors       bool              `yaml:"skipJsErrors,omitempty" json:"skipJsErrors"`
+	QuarantineMode     bool              `yaml:"quarantineMode,omitempty" json:"quarantineMode"`
+	SkipUncaughtErrors bool              `yaml:"skipUncaughtErrors,omitempty" json:"skipUncaughtErrors"`
+	SelectorTimeout    int               `yaml:"selectorTimeout,omitempty" json:"selectorTimeout"`
+	AssertionTimeout   int               `yaml:"assertionTimeout,omitempty" json:"assertionTimeout"`
+	PageLoadTimeout    int               `yaml:"pageLoadTimeout,omitempty" json:"pageLoadTimeout"`
+	Speed              float32           `yaml:"speed,omitempty" json:"speed"`
```

Comment:
I'd recommend to just use `float`.
```suggestion
	Speed              float           `yaml:"speed,omitempty" json:"speed"`
```

Reply:
update to `float64` since there is no `float` actually 😉